mlflow/utils/databricks_utils.py

```diff
@@ -439,34 +475,65 @@ def get_databricks_host_creds(server_uri=None):
         MlflowHostCreds which includes the hostname and authentication information necessary to
         talk to the Databricks server.
     """
+    # Since we do not record OAuth expiration time in OAuth file, perform periodic refresh
+    # of OAuth cache here. As currently configured (02/24) OAuth token in model serving environment
```

Comment:
minor nit: should we remove the date?

Reply:
Just added the date in case this is out of sync later and we change the refresh interval in serving scheduler